sources/index.html

```diff
@@ -51,5 +51,179 @@ <h2>I'm a software developer</h2>
+                <article class="project-list">
+                    <div class="project1">
+                        <img src="../images/ImgPlaholder.png" alt="project-preview">
+                        <h2>Multi-Post Stories<br>
+                        Gain+Glory</h2>
+                        <ul class="button-grid">
+                            <li class="lang-button1">Ruby on rails</li>
+                            <li class="lang-button2">Css</li>
+                            <li class="lang-button3">JavScript</li>
+                            <li class="lang-button4">Html</li>
+                        </ul>
+                        <section class="main-button">
+                            <button class="projectview" id="test3">See Project</button>
+                        </section>
+                    </div>
+                </article>
+                <!--project 5-->
+                <article class="project-list">
+                    <div class="project1">
+                        <img src="../images/ImgPlaholder.png" alt="project-preview">
+                        <h2>Multi-Post Stories<br>
+                        Gain+Glory</h2>
+                        <ul class="button-grid">
+                            <li class="lang-button1">Ruby on rails</li>
+                            <li class="lang-button2">CSS</li>
+                            <li class="lang-button3">JavaScript</li>
+                            <li class="lang-button4">HTML</li>
+                        </ul>
+                        <section class="main-button">
+                            <button class="projectview" id="test4">See Project</button>
+                        </section>
+                    </div>
+                </article>
+                <!--Project 6-->
+                <article class="project-list">
+                    <div class="project1">
+                        <img src="../images/ImgPlaholder.png" alt="project-preview">
+                        <h2>Multi-Post Stories<br>
+                        Gain+Glory</h2>
+                        <ul class="button-grid">
+                            <li class="lang-button1">Ruby on rails</li>
+                            <li class="lang-button2">Css</li>
+                            <li class="lang-button3">JavaScript</li>
+                            <li class="lang-button4">HTML</li>
+                        </ul>
+                        <section class="main-button">
+                            <button class="projectview" id="test5">See Project</button>
+                        </section>
+                    </div>
+                </article>
+            </div>
+        </section>
+        <!--about page-->
+        <section class="about">
+            <div class="aboutme-desc">
+                <h2>About me</h2>
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
```

Comment:
- [ ] [OPTIONAL]   Background image needs to be adjusted a little bit to match the design, currently, it's almost overlapping with text and you could see at Figma design it starts next to the `About me` Title. If you tried shrinking it to another mobile view less than 370px  wide it will overlap with the text, could you fix that, please? 

Yours | Figma design
:--:|:--:
![image](https://user-images.githubusercontent.com/87689547/148614712-8281c528-1179-446e-a83e-50cf252e914c.png) | ![image](https://user-images.githubusercontent.com/87689547/148614379-1467fd68-fd2d-4c0d-af1e-5c4092b2eea4.png)